frontend/src/services/Api.ts

```diff
@@ -567,18 +609,27 @@ export const getTrace = (idTrace: string) => {
   return newRequest<TracingSingleResponse>(HTTP_VERBS.GET, urls.tracingTrace(idTrace), {}, {});
 };
 
-export const getGraphElements = (params: any) => {
-  return newRequest<GraphDefinition>(HTTP_VERBS.GET, urls.namespacesGraphElements, params, {});
+export const getGraphElements = (params: GraphElementsQuery, cluster?: string) => {
```

Comment:
I don't think this signature should be exported.  The `cluster?` param should not exist here, I think it's here only because you want to call this from `getNodeGraphElements`.  I'd suggest you make this a private function and introduce a thin exported function that does not take the `cluster?` param, and just then calls the private function.

Reply:
Ok, so the default getGraphElements method is cluster-wide. In that case I will remove the cluster param from the method.